src/main/java/org/apache/accumulo/access/Authorizations.java

```diff
@@ -58,9 +59,12 @@ public int hashCode() {
     return authorizations.hashCode();
   }
 
+  /**
+   * @return a String containing the sorted, comma-separated set of authorizations
+   */
   @Override
   public String toString() {
-    return authorizations.toString();
+    return new TreeSet<>(authorizations).toString();
```

Comment:
I wonder what the penalty is here when the user doesn't care about sorting? Consider the example:
```
  LOG.info("Authorizations: {}", authorizations);
```

I wonder if we should leave `toString()` the way that it was, and add a new method with this implementation called `toSortedString` or something. i'm usually on the fence regarding changes like this because we are baking in a performance penalty 100% of the time for something that we *think* is a good idea. I'd rather let the user choose if they want that penalty.

Reply:
I'm fine with that. Although in that case do you think its worth even creating a new method like `toSortedString` or should we just let the user use `toSet` and sort and create the string on their own?

Reply:
I'm fine with letting users do it on their own. We should get the issue originator to weigh in on this.

Reply:
@ctubbsii do you have any input on this as the creator of the issue associated with this PR?

Reply:
I think now that Authorizations.of() takes a set, we could just use the set that the user passed in, preserving whatever order it already had. If it's an immutable set already, we can just keep a reference to the same object instead of doing an unnecessary copy. If it's not, we can preserve the order when we do the copy (maybe using `LinkedHashSet.addAll()` wrapped with `Collections.unmodifiableSet`?). The only issue with this approach is that `asSet()` won't return something that has the `SortedSet` interface if it originally was a mutable sorted set, like TreeSet, but that might be an acceptable compromise).